tests/testsuite.c

```diff
@@ -103,10 +103,11 @@ int TestsuiteTest(int argc, char** argv)
 
     WSTARTTCP();
 
-    wolfSSH_Init();
     #if defined(DEBUG_WOLFSSH)
         wolfSSH_Debugging_ON();
     #endif
+
+    wolfSSH_Init();
```

Comment:
Specifically enabling logging before calling wolfSSL_Init, since that does operations that may need logged.